apps/frontend/src/pages/onboarding/libs/enums/enums.ts

```diff
@@ -0,0 +1,7 @@
+const NO_MAGIC = {
```

Comment:
I think it's a bad idea to have it in this file, maybe move it to a separate file? And maybe it is better to change the name so that it is more clear what these numbers are and what they are for